basic/src/test/java/org/jboss/hal/testsuite/test/configuration/requestcontroller/RequestControllerSubsystemTestCase.java

```diff
@@ -59,6 +59,7 @@ public static void afterClass() throws InterruptedException, TimeoutException, I
         }
     }
 
+    @Category(Standalone.class)
     @Test
     public void verifyActiveRequests() throws Exception {
```

Comment:
Maybe I am not looking everywhere but I do not see a way how to see active request on server in web console gui. That looks like a possible bug.